### DISCUSSION TITLE
403 Forbidden Error with Asaas Webhook and Edge Functions

### DISCUSSION THREAD

**Discussion by pedrohssales** [supabase/supabase#21981] [Questions]
Hello, Supabase community,

I'm encountering an issue when trying to receive webhooks from Asaas in a Supabase Edge Function. The function is called 'asaas_webhook'.

When I send POST requests to the endpoint using Postman, everything works perfectly, even with repeated IDs. However, when Asaas sends a webhook to the same endpoint, it returns a 403 Forbidden error. To confirm that Asaas is sending the webhook correctly, I set up a test endpoint on Make, which received the webhook without any issues.

I have checked for any network restrictions or firewall settings that might be blocking the requests, but I haven't found anything that could be causing this error on my side. I suspect there might be some configuration or restriction in Supabase Edge Functions that is causing this issue.

Has anyone else experienced something similar or have any suggestions on what might be causing this 403 Forbidden error? Any help or guidance would be greatly appreciated.

Thank you!

**Comment by danielkv** [comment]
I'm having the same problem.
I saw you newer issue https://github.com/supabase/supabase-js/issues/987. It seems that no one from supabase team has a clue for that. It doesn't look to be something to hard to fix. @awalias Can you help us?
